packages/@lightningjs/ui-components/src/components/Tile/Tile.js

```diff
@@ -175,6 +182,26 @@ export default class Tile extends Surface {
     );
   }
 
+  /* ------------------------------ Icon ------------------------------ */
+
+  _updateIcon() {
+    const iconObject = {
+      w: this.style.iconWidth,
+      h: this.style.iconHeight,
+      icon: this.iconSrc,
+      alpha: this._isFocusedMode ? 1 : 0.01,
+      x: this.style.iconX,
+      y:
+        this.metadataLocation === 'inset'
+          ? this.style.iconYInset
+          : this.style.iconYStandard
```

Comment:
Another way I believe you could set the `y` position of the icon would be to calculate its position based on where the MetaData is and subtract by its height to position the icon above it. 
```suggestion
      mountY: this._isInsetMetadata ? 1 : 0,
      y: this._isInsetMetadata
        ? this._metadataY - (this._Metadata ? this._Metadata.h : 0)
        : this.style.iconYStandard
```
By doing it like this, it should also handle if the Tile's height were to increase. The provider logo will always position itself using Metadata's positioning.

Reply:
You are right, Thanks for suggestion. I will update that now.